src/Resources/ResourceFactory.php

```diff
@@ -195,6 +196,54 @@ protected function serviceFactory(array &$properties)
             unset($properties['attributes']['transit_time']['max']);
         }
 
+        if (isset($properties['attributes']['regions_from'][0])) {
+            $regionFromAttribute = $properties['attributes']['regions_from'][0];
+
+            $countryCode = $regionFromAttribute['country_code'];
+            $regionCode = $regionFromAttribute['region_code'];
+
+            /** @var RegionInterface $regionFrom */
+            $regionFrom = $this->api->getRegions([
+                    'country_code' => $countryCode,
+                    'region_code'  => $regionCode,
+                ]
+            )->get()[0];
+
+            if(!$regionFrom){
+                throw new ResourceNotFoundException(
+                    'No region found for country code '.$countryCode.' and region code '. $regionCode
```

Comment:
If `$regionCode` is `null`, the sentence will end in: "and region code ". Maybe we can stringify it if it is `null` to indicate that there was no region code.